test/spec/functions.ts

```diff
@@ -33,7 +33,7 @@ export const testFnByName: Record<string, (data: any) => any> = {
  * ```
  */
 function aggregate(input: string[]): string | null {
-  const agg = aggregateSignatures(input.map((hex) => Signature.deserialize(fromHex(hex))));
+  const agg = aggregateSignatures(input.map(fromHex));
```

Comment:
I wanted to keep the tests as pure as possible to make sure the library meets the spec completely. In some tests there are bad keys and signatures and those can potentially throw errors in the `deserialize` method, which would require JS catch for the error case when the spec calls for false to be returned instead of the error.